packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -77,47 +77,6 @@ class TextSelectionPoint {
   }
 }
 
-// Check if the given code unit is a white space or separator
```

Comment:
This file's diff is really hard to read, but basically what happened was:

  * Lots of methods moved to TextEditingAction and TextEditingValue.
  * Some alphabetizing/reordering may have happened...
  * TextMetrics was implemented.